plugins/layout.ts

```diff
@@ -27,7 +27,13 @@ function layoutTag(
 
   const [_, file, data] = match;
 
-  const varname = "__content";
+  let varname
+  if (output.startsWith("__layout")) {
+    varname = output + "_layout";
+  } else {
+    varname = "__layout";
+  }
+
```

Comment:
Just a small thing. I think it's better to do this:

```js
const varname = output.startsWith("__layout")
  ? output + "_layout"
  : "__layout";
```

to avoid to create the variable with let and assign the value later.